backend/bitboxbase/mdns/detector.go

```diff
@@ -75,6 +77,14 @@ func (detector *Detector) TryMakeNewBase(ip string) (bool, error) {
 	}
 	var baseDevice bitboxbase.Interface
 	var err error
+
+	match := regexp.MustCompile(`[0-9]+(?:\.[0-9]+){3}(:[0-9]+)`).MatchString(ip)
```

Comment:
Thinking about it, why do you have IP everywhere instead of `address`, which could also be a hostname? I think it will be better to use `address` everywhere, as http.Get() etc. can also use that.